rails-commenting.rb

```diff
@@ -33,14 +33,14 @@ def create
     end
   end
 
-  # ---6)
+  # ---6) The edit method allows for the user to search for a certain id and make changes to that id.
```

Comment:
New and Edit share the quirk where they are specifically associated with forms for the user to input data